roles/set_scale/tasks/main.yml

```diff
@@ -0,0 +1,76 @@
+# See meta/main.yml for a list of dependency roles that will be executed prior to this role
+---
+- name: Ensure parameters have been set
+  fail: msg="Bailing out. This role requires 'machineset_instance_type' and 'scale' to be defined"
+  when: not machineset_instance_type or not scale
```

Comment:
I wonder what happens when `scale = 0`, did you try?

Reply:
Good point. Just tested - even when scale is `0`, seems like it's actually being treated as `"0"` which is a non empty string and thus `not scale` is still negative.